src/query_tracker.py

```diff
@@ -0,0 +1,76 @@
+"""
+Localized account of all Queries related to this project's main functionality
```

Comment:
Only opinion of this that I have is the name of the file `query_tracker.py` which gives the hint of something being tracked where as this is only a data class for holding query information in one place. What do you think about this?

Reply:
Do you have a suggested name alternative for this? Happy to change it. Shorter would be nice. We could call it queries.py I just didn't want to put it in constants because that file is already a mess that needs to be dealt with.

Reply:
Changed to `queries.py`